Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contest/Contest.tsx

```diff
@@ -83,19 +84,17 @@ const Contest = () => {
 
     const renderTimeRemaining = useCallback(
         () => {
-            const { endTime } = contest || {};
-
-            if (!endTime) {
+            if (!remainingTimeInMilliseconds) {
                 return null;
             }
 
-            const duration = convertToSecondsRemaining(new Date(endTime));
+            const duration = convertToSecondsRemaining(new Date(remainingTimeInMilliseconds));
```

Comment:
Are you sure this is correct? Looking at the implementation of `convertToSecondsRemaining()`, it calculates the remaining time to the provided `date`, that is expected to be in the future.

If my understanding is correct, remaining seconds will be a very small time in respect of dates, i.e. 3 days, or 4 days, or a week (7 days). 7 days = 7*24 hours = 7*24*60 minutes = 7*24*60*60 seconds = 7*24*60*60*100 milliseconds =  60480000 milliseconds.
`new Date(60480000) = Thu Jan 01 1970 18:48:00 GMT+0200`.

In your case, since you already have the `remaining time` in milliseconds, you can just devide it by `100`.